Solutions/Marain.Workflow.Specs/Bindings/WorkflowSqlBindings.cs

```diff
@@ -32,22 +34,61 @@ public static class WorkflowSqlBindings
         public static void SetupDatabases(FeatureContext featureContext)
         {
             IServiceProvider serviceProvider = ContainerBindings.GetServiceProvider(featureContext);
-            _ = serviceProvider.GetRequiredService<ITenantSqlConnectionFactory>();
+            ITenantSqlConnectionFactory sqlConnectionFactory = serviceProvider.GetRequiredService<ITenantSqlConnectionFactory>();
+            ITenantCosmosContainerFactory factory = serviceProvider.GetRequiredService<ITenantCosmosContainerFactory>();
             ITenantProvider tenantProvider = serviceProvider.GetRequiredService<ITenantProvider>();
 
             SqlConfiguration config = tenantProvider.Root.GetDefaultSqlConfiguration();
-            config.DisableTenantIdPrefix = true;
 
+            string containerBase = Guid.NewGuid().ToString();
+
+            config.ConnectionString = "Server=(localdb)\\mssqllocaldb;Trusted_Connection=True;MultipleActiveResultSets=true";
+            config.Database = $"workflow-{containerBase}";
+            config.IsLocalDatabase = true;
+            config.DisableTenantIdPrefix = true;
             tenantProvider.Root.SetDefaultSqlConfiguration(config);
+
+            CosmosConfiguration cosmosConfig = tenantProvider.Root.GetDefaultCosmosConfiguration();
+            cosmosConfig.DatabaseName = "endjinspecssharedthroughput";
+            cosmosConfig.DisableTenantIdPrefix = true;
+            tenantProvider.Root.SetDefaultCosmosConfiguration(cosmosConfig);
```

Comment:
Should these Cosmos bits still be here?

Reply:
Yes. Because the test document store uses cosmos.